packages/grafana-data/src/field/fieldColor.ts

```diff
@@ -32,55 +32,47 @@ export const fieldColorModeRegistry = new Registry<FieldColorMode>(() => {
         };
       },
     },
-    new FieldColorSchemeMode({
-      id: FieldColorModeId.PaletteSaturated,
-      name: 'Color by series / Saturated palette',
-      //description: 'Assigns color based on series or field index',
-      isContinuous: false,
-      isByValue: false,
-      colors: [
-        'blue',
-        'red',
-        'green',
-        'yellow',
-        'purple',
-        'orange',
-        'dark-blue',
-        'dark-red',
-        'dark-yellow',
-        'dark-purple',
-        'dark-orange',
-      ],
-    }),
+    // new FieldColorSchemeMode({
```

Comment:
nit: maybe remove this?

Reply:
yea, just want to add it back soon, just the order and the colors need to be figured out :)